internal/checkout/source_provider.go

```
@@ -218,6 +218,61 @@ func (cfg *Config) validate() error {
 	return nil
 }
 
+func (cfg *Config) writeActionOutputs(cli *git.GitCLI) error {
+	//Output commit details
+	outDir := os.Getenv("CLOUDBEES_OUTPUTS")
+	err := os.WriteFile(filepath.Join(outDir, "provider"), []byte(cfg.Provider), 0640)
+	if err != nil {
+		return err
+	}
+	fullUrl, err := cfg.fetchURL(cfg.SSHKey != "")
+	if err != nil {
+		return err
+	}
+	err = os.WriteFile(filepath.Join(outDir, "repository-url"), []byte(fullUrl), 0640)
+	if err != nil {
+		return err
+	}
+	commitId := cfg.Commit
+	if commitId == "" {
+		commitId, _ = cli.GetLastCommitId()
```

Why is the error being ignored here?